Update site content

aarushivenk opened the pull request:
# Pull Request

## Description

Updating outdated site content and fixed basic issues in the external site. Updated the readme with details on local development setup to run site locally.

aarushivenk commented on docs/components/tags.md:
```diff
@@ -3,7 +3,7 @@ status: "stable"
 last_updated: "2025-07-09"
 ---
 
-# Tags Guidance
```
Guidance seemed redundant here

aarushivenk commented on mkdocs.yml:
```diff
@@ -1,9 +1,39 @@
+        icon: material/brightness-4
+        name: Switch to system preference
+
+  font:
```
also added font

aarushivenk commented on docs/branding/approach-to-ai.md:
```diff
@@ -56,13 +56,13 @@ Chatbot interface for users to ask questions about a case within Case Management
-> **Helpful Tips**
->
-> If an icon is needed and cannot be an image, the "magic" icon can be used
+!!! info "Helpful Tips"
```
to confirm are you viewing this from the markdown view in github or locally? 

mkdocs material has their own way of doing callouts that's a little different than standard markdown format: https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=admo

here's what this looks like when I was testing it out:
<img width="1379" height="742" alt="image" src="https://github.com/user-attachments/assets/e3eba25e-d118-4be0-abfc-7797493a40cc" />

aarushivenk commented on docs/branding/icons.md:
```diff
@@ -11,84 +11,84 @@ The following icons are available for use in Appian Solutions. Icons are organiz
-| Vendors | 🏪 | store | Object |
-| Warning | ⚠️ | exclamation-triangle | Banner |
-| Workflow Template | ✓ | tasks-alt | Object |
+| Accounts | :fontawesome-solid-suitcase: | suitcase | Object |
```
In the mkdocs.yml file I was able to add in 

"icon:
    repo: fontawesome/brands # Default repository for brand icons"
    
 which allows us to render fontawesome icons on the site. 
 
 The icons don't render in the github file preview though.

Included screenshot of what this looks like on the site: 

<img width="1270" height="951" alt="image" src="https://github.com/user-attachments/assets/6e16d09b-cf83-459e-86f5-4d68b246c8af" />

pglevy replied:
oh, my bad, I forgot I didn't go through the mkdocs build 🤦‍♂️

aarushivenk commented on docs/branding/approach-to-ai.md:
```diff
@@ -56,13 +56,13 @@ Chatbot interface for users to ask questions about a case within Case Management
-> Alt-text is not needed as the image is entirely decorative
+    If an icon is needed and cannot be an image, the "magic" icon can be used
+
+!!! abstract "Accessibility"
```
including an image of how this is displaying for me locally

<img width="718" height="663" alt="image" src="https://github.com/user-attachments/assets/572307b3-6ab0-4f44-9217-f3097366e7ad" />

aarushivenk commented on docs/components/tabs.md:
```diff
@@ -19,8 +19,9 @@ Tabs are used to navigate between alternate views within a user interface
 
-> [!Note]
-> **Accessibility:** The platform's record tabs have accessibility features available out of the box — nothing more is necessary.
+!!! abstract "Accessibility"
```
<img width="709" height="674" alt="image" src="https://github.com/user-attachments/assets/de0a8b89-d3b9-4b64-85a9-461070a3dfd4" />

aarushivenk commented on docs/branding/typography.md:
```diff
@@ -47,7 +29,7 @@ When placing an action component aligned to the other end of the header, place t
 ### Option 2 of 2: Use Side By Side Layout
 
-When you have patterns that require information or interactions immediately alongside the header, place the *a!richTextHeader* section title into an *a!sideBySideLayout*. Note that you cannot place an *a!sectionLayout* header into an *a!sideBySideLayout*.
+When you have patterns that require information or interactions immediately alongside the header, place the *a!headingField* section title into an *a!sideBySideLayout*. Note that you cannot place an *a!sectionLayout* header into an *a!sideBySideLayout*.
```
oh hmm good catch, I can update these

pglevy replied:
I guess it makes sense for values too (e.g., `SOLID` OR `OUTLINE`). Maybe anything that is "literal" as you would enter it in an expression editor?

pglevy commented:
@aarushivenk, is there a requirements.txt file you have locally that needs to be committed?

aarushivenk commented:
> @aarushivenk, is there a requirements.txt file you have locally that needs to be committed?

no, I was trying it out but saw it's still in the readme! 

that line should be "pip install mkdocs-material" instead, I'll update the readme

pglevy commented:
> > @aarushivenk, is there a requirements.txt file you have locally that needs to be committed?
> 
> no, I was trying it out but saw it's still in the readme!
> 
> that line should be "pip install mkdocs-material" instead, I'll update the readme

thanks! all is well!

pglevy commented:
Also noticed this (`[Branding Images](../assets/images/logo-and-favicon`) is returning a 404 in `branding/logo-and-favicon.md`. Along with the other AI imagery one I mentioned, it shows up in the build messages when you run `mkdocs serve`.

aarushivenk commented:
> Also noticed this (`[Branding Images](../assets/images/logo-and-favicon`) is returning a 404 in `branding/logo-and-favicon.md`. Along with the other AI imagery one I mentioned, it shows up in the build messages when you run `mkdocs serve`.

addressed in latest two commits! the 404 errors were because we had links pointing to folders rather than files 

updated this as well as your other comments
